update-po-files.sh

```diff
@@ -88,7 +88,7 @@ echo "LANG;TRANSLATED;FUZZY;UNTRANSLATED" > "${CSVFILE}"
 for i in $LINGUAS
 do
   echo "## $i"
-  msgmerge --force-po $LOCALDIR/$i/kicad.po $LOCALDIR/kicad.pot -o $LOCALDIR/$i/kicad.po 2> /dev/null
+  msgmerge -N --force-po $LOCALDIR/$i/kicad.po $LOCALDIR/kicad.pot -o $LOCALDIR/$i/kicad.po 2> /dev/null
```

Comment:
What does this mean and why? What is a similar string?

Reply:
Read above and also my reply... "it stops msgmerge from guessing, which it usually does wrong" and I replied: "Say developers corrected a small typo in source string (not so uncommon)... is it not useful to leave automatic fuzzy generation as it is?" that means that I would have left the -N option...

Reply:
Problem is that it messed up a lot of translations.

For example, in Bulgarian, "Move alias up" and "Move End Point" both translate to "Премести извод" (move terminal), and "Green 3" translates to "Гръцки", because when "Green 3" was introduced as a string, the nearest similar string was "Greek".

Translators can always run msgmerge with fuzzy generation if they want to, but most of the time, it is more work than it is good for. It also breaks the "msgen" invocation, so I'd like to have it disabled at least for the "en" locale.

Reply:
The thing is that it does not mess up correctly translated strings but only those whose reference is changed somehow, even slightly as for my example. Deleting incorrect fuzzied strings is really not a big deal but having to rewrite completely translated strings because the source introduced a comma is a bit annoying though...

Reply:
Yes, I can see that, so probably sanest to use that only for "en" then.